samples/tfm_integration/tfm_secure_partition/dummy_partition/CMakeLists.txt

```diff
@@ -0,0 +1,54 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+add_library(tfm_app_rot_partition_dp STATIC)
```

Comment:
Cleaner to omit the target_sources invocations below and place the sources straight into the add_library invocation.

Reply:
This follows TF-Ms general pattern for adding partition sources. I prefer to stick to it for consistency for that reason.